src/typings/tacoLiveReload.d.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+
+/// <reference path="../typings/Q.d.ts" />
+
+declare module TacoLiveReload {
+
+    export function start(projectRoot: string, platforms: string[], options: LiveReloadOptions): Q.Promise<any>;
+    export function on(eventName: string, cb: (...args : any[]) => void): void;
+    export function isLiveReloadActive(): boolean;
+
+    export class Patcher {
+        constructor(projectRoot: string, platforms: string[]);
+        public removeCSP(): void;
+        public patch(serverURL: string): Q.Promise<any>;
+    }
+
+    export interface LiveReloadHandle {
+        reloadFile(file?: string): Q.Promise<any>;
```

Comment:
- The 'file' argument is the file that we want to reload.
- In a multipage application, if possible, diffing of the file occurs and only changed parts will be reconstructed on the browser-side (e.g: .css). Otherwise, the full application gets reloaded (e.g: .js, .html, others).
- This handle will only be available after the app has loaded and started, so there's no risk of utilizing uninitialized resources.